src/test/java/com/microsoft/azure/relay/HybridConnectionListenerTest.java

```diff
@@ -417,7 +417,7 @@ public void closedOrReadonlyResponseTest() throws IOException {
 				response.getHeaders().put(headerKey, headerVal);
 				modifiedAfterClose.complete(null);
 			} catch (Exception e) {
-				modifiedAfterClose.completeExceptionally(e);
+				modifiedAfterClose.completeExceptionally(e.getCause());
```

Comment:
>e.getCause() [](start = 45, length = 12)

The exception was changed to IOException in the last check-in and it's wrapped by RuntimeException now